javaagent-tooling/src/main/java/io/opentelemetry/javaagent/tooling/InstrumentationModule.java

```diff
@@ -251,6 +252,18 @@ private String mainInstrumentationName() {
     return instrumentationNames.iterator().next();
   }
 
+  /**
+   * Instrumentation modules can override this method to specify additional packages (or classes)
+   * that should be treated as "library instrumentation" packages. Classes from those packages will
+   * be treated by muzzle as instrumentation helper classes: they will be scanned for references and
+   * automatically injected into the application classloader if they're used in any type
+   * instrumentation. The class name predicate returned by this method is added to the default ones
+   * defined in {@link InstrumentationClassPredicate}.
+   */
+  public Predicate<String> additionalLibraryInstrumentationPackage() {
```

Comment:
Please describe in more details what String is being passed to this predicate.

Reply:
a thought, not sure, on trying to make it clearer
```suggestion
  public boolean isHelperClass(String className) {
    return false;
  }
```

Reply:
Good point 👍 

It's a good chance to learn how method references are represented in the bytecode 😂